.eslintrc.json

```diff
@@ -17,7 +17,8 @@
 	  "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
 	  "import/no-unresolved": "off",
 	  "no-shadow": "off",
-	  "arrow-parens":"off",
-	  "no-restricted-syntax": ["error", "FunctionExpression", "WithStatement", "BinaryExpression[operator='in']"]
+	  "arrow-parens": "off",
+	  "no-restricted-syntax": ["error", "FunctionExpression", "WithStatement", "BinaryExpression[operator='in']"],
+	  "react/prop-types": 0
```

Comment:
As the previous reviewer mentioned, you can check out https://reactjs.org/docs/typechecking-with-proptypes.html to typecheck your props
That is;
```
Book.propTypes = {
  book: PropTypes.......
}
```